marklogic-client-api/src/main/java/com/marklogic/client/DatabaseClientBuilder.java

```diff
@@ -158,6 +158,12 @@ public DatabaseClientBuilder withCertificateAuth(String file, String password) {
 			.withCertificatePassword(password);
 	}
 
+	public DatabaseClientBuilder withCertificateAuth(SSLContext sslContext, X509TrustManager trustManager) {
```

Comment:
This was not needed for the fix, it's just some syntactic sugar for the use case of doing certificate auth with a user-provided SSLContext instead of a certificate file path.